docs/ref-release-notes.md

```diff
@@ -12,11 +12,15 @@
 
 # Changed
```

Comment:
I really really dislike this format @darahayes 
This is very vague and unredable. When I created it initially I was hoping to get something like: https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript

In most of the cases people will be looking into introduction for new features. This format do not support long descriptions and large migration text etc.

Reply:
If it's not working, let's change it.

Reply:
I want to discuss about it or see examples because I'm probably missing the concept. Do you still have website that explains it ?